modular_skyrat/modules/reagent_forging/code/forge_weapons.dm

```diff
@@ -240,3 +240,68 @@
 /obj/item/shovel/reagent_weapon/Initialize(mapload)
 	. = ..()
 	AddComponent(/datum/component/reagent_weapon)
+
+/obj/item/ammo_casing/caseless/arrow/wood/forged
+	desc = "An arrow made of wood, typically fired from a bow. It can be reinforced with sinew."
+	projectile_type = /obj/projectile/bullet/reusable/arrow/wood/forged
+
+/obj/item/ammo_casing/caseless/arrow/wood/forged/attackby(obj/item/attacking_item, mob/user, params)
+	if(istype(attacking_item, /obj/item/stack/sheet/sinew))
+		var/obj/item/stack/stack_item = attacking_item
```

Comment:
you can remove this var declaration and change the type assigned on `attackby`

Reply:
I think I would prefer to keep it like this in case I want to add more.